bridge-stp.in

```diff
@@ -14,7 +15,7 @@
 # (possibly in a background job after a delay; see the comments in the code
 # below).  No further configuration is performed automatically by this script at
 # that time.  Additional configuration is usually performed by
-# @ifupdownfile@ (which calls `brctl stp <bridge> on`
+# @ifupdownfile@ (which calls `ip ... stp_state 1`
```

Comment:
Since you list it in full elsewhere, why not use `ip link set <bridge> type bridge stp_state 1` here too?